Portal/src/Datahub.Portal/Pages/Workspace/WebApp/WorkspaceWebAppPage.razor

```diff
@@ -0,0 +1,184 @@
+﻿@using Datahub.Application.Services.Security
+@using Datahub.Portal.Views.Dialogs
+@using Datahub.Shared.Entities
+@using Microsoft.Identity.Web
+@using System.Text.Json.Nodes
+@using Datahub.Infrastructure.Queues.Messages
+@using MediatR
+
+@inject IDbContextFactory<DatahubProjectDBContext> _dbContextFactory
+@inject IKeyVaultUserService _keyVaultUserService
+@inject IDialogService _dialogService
+@inject ISnackbar _snackbar
+@inject ILogger<WorkspaceWebAppPage> _logger
+@inject MicrosoftIdentityConsentAndConditionalAccessHandler _consentHandler
+@inject IMediator _mediatr
+
+<MudStack>
+    <MudStack Row="true">
+        <MudText Typo="Typo.h1">
+            @Localizer["Web Application"]
+        </MudText>
+        <MudSpacer/>
+        <MudIconButton Icon="@Icons.Material.Filled.Settings" Color="Color.Dark" Size="Size.Medium" OnClick="async () => { await ConfigureWebApp(); }"/>
+    </MudStack>
+
+    <MudText>
+        @Localizer["Here you can manage your web application. You can view and edit the configurations of your web application"]
+    </MudText>
+
+    <MudText Typo="Typo.h2">
+        @Localizer["Web Application Info"]
+    </MudText>
+    <MudText>
+        @Localizer["Host: "] <MudLink Href="@_webAppHost">@_webAppHost</MudLink>
+    </MudText>
+    <MudText>
+        @Localizer["Id: "] <code>@_webAppId</code>
+    </MudText>
+    <MudText>
+        @Localizer["Status: "]
+    </MudText>
+    @if (_isConfigured)
+    {
+        <MudText>
+            @Localizer["Framework: "] <code>@_webAppFramework</code>
+        </MudText>
+        <MudText>
+            @Localizer["Git repo: "] <code>@_webAppRepo</code>
+        </MudText>
+        <MudText>
+            @Localizer["Compose path: "] <code>@_webAppCompose</code>
+        </MudText>
+    }
+    else
+    {
+        <MudText>
+            @Localizer["This web application is not configured yet. Please configure it using the settings icon in the top right corner of the section."]
+        </MudText>
+    }
+
+</MudStack>
+
+@code {
+
+    [Parameter] public string WorkspaceAcronym { get; set; }
+
+    private string _webAppHost = "<web_app_host>";
+    private string _webAppId = "<web_app_id>";
+    private string _webAppFramework = "<framework>";
+    private string _webAppRepo = "<git_repo>";
+    private string _webAppCompose = "<path_to_compose>";
+
+    private bool _isConfigured;
+
+    protected override async Task OnInitializedAsync()
+    {
+        await base.OnInitializedAsync();
+
+        try
+        {
+            await HydrateValues();
+        }
+        catch (MicrosoftIdentityWebChallengeUserException e)
+        {
+            _consentHandler.HandleException(e);
```

Comment:
Which scope are you adding here?

Reply:
Its just an artifact of "copilot, create a web app page based and inspire yourself from the database page"